translations/tiled_pt.ts

```diff
@@ -309,7 +309,7 @@
     <message>
         <location filename="../src/plugins/csv/csvplugin.cpp" line="+166"/>
         <source>CSV files (*.csv)</source>
-        <translation>Ficheiros CSV (*.csv)</translation>
```

Comment:
For Tiled 1.11,  @SantosSi had changed all these the other way around in cfd589bbcb1c81278de1a481c2c9006ac66b81d1. We should try to avoid going back and forth on this, so maybe you could agree on a preferred translation?

Reply:
Or maybe it's not clear that https://hosted.weblate.org/projects/tiled/translations/pt_PT/ is the "Portuguese (Portugal)" translation whereas https://hosted.weblate.org/projects/tiled/translations/pt/ is actually "Portuguese (Brazil)" (just, due to leaving out the `_BR`, also serves as fallback for strings without a specific translation for Portugal?

Reply:
> For Tiled 1.11, @SantosSi had changed all these the other way around in [cfd589b](https://github.com/mapeditor/tiled/commit/cfd589bbcb1c81278de1a481c2c9006ac66b81d1). We should try to avoid going back and forth on this, so maybe you could agree on a preferred translation?

"Ficheiro" is not a word used in Brazilian Portuguese (Brazil - pt_BR). It is a word used in the Portuguese language of Portugal (pt_PT). In Brazilian Portuguese, the word "Arquivo" ("File" in English) is used.